packages/marshaller/src/ddl2/dashboard.ts

```diff
@@ -172,24 +246,43 @@ export class Dashboard extends ChartPanel {
                 d3Text("https://raw.githubusercontent.com/hpcc-systems/Visualization/master/utils/data/data/carriers.csv"),
                 d3Text("https://raw.githubusercontent.com/hpcc-systems/Visualization/master/utils/data/data/stats.csv")
             ]).then(([airports, carriers, stats]) => {
-                const airportsElement = this.addDatabomb("airports", airports, "csv", new Project().computedFields([
-                    new Project.ComputedField().label("Code").type("=").column1("code"),
-                    new Project.ComputedField().label("Airport").type("=").column1("name"),
-                    new Project.ComputedField().label("Count").type("scale").column1("count").constValue(1)
-                ]), new Sort().column([new Sort.Column().fieldID("Count").descending(true)]));
+                const popupElement = this.addDatabomb("popup", '[{ "Airport": "", "Airline": "" }]', "json");
```

Comment:
It seems like these samples belong in the `dashy.ts` layer

Reply:
I agree - but I also plan on removing them once we go gold...